triage/interactive.js

```diff
@@ -303,7 +303,7 @@ function getData() {
     setElementVisibility('btn-sig-group', false);
   }
 
-  var url = '/k8s-gubernator/triage/';
+  var url = '/triage/';
```

Comment:
Not `/k8s-triage/triage` ?

Reply:
I thought it was for the benefit of running locally, but it didn't seem to make a difference either way.  It could probably be `url=''` but I was mainly interested in getting the bucket name out of the source